process/scToProtocol/stakingToPeer.go

```diff
@@ -197,27 +174,6 @@ func (stp *stakingToPeer) UpdateProtocol(body block.Body, nonce uint64) error {
 	return nil
 }
 
-func (stp *stakingToPeer) peerUnregistered(account *state.PeerAccount, nonce uint64) error {
```

Comment:
Were these removed because we don't need them now or we don't need them at all?

Reply:
no more need for them. As we do not create peerchanges. It is directly saved in peer state. PeerChanges were a must when those were processed by shards as well.

Reply:
ok